packages/scalecube-microservice/src/helpers/types.ts

```diff
@@ -29,10 +29,7 @@ export interface GetProxyOptions {
   serviceDefinition: MicroserviceApi.ServiceDefinition;
 }
 
-export interface Qualifier {
-  serviceName: string;
-  methodName: string;
-}
+export const { Qualifier } = MicroserviceApi.Qualifer;
```

Comment:
why you need to add the qualifier to the types. just use it from the api package